### ISSUE TITLE
Log initialization

### ISSUE THREAD

**Issue by ReliableLion** [netgroup-polito/frog4-sdn-do#21]
After this commit [62f83d6](https://github.com/netgroup-polito/frog4-sdn-do/commit/f7ba999ccddfd66c1b1379bac423aa4d3bf094e5) the sdn-do is not able to create and initialize a log file. I tried to move the load_configuration() method back to the config.py and it worked.

**Comment by frisso** [comment]
@ReliableLion: Is this issue still open?

**Comment by ReliableLion** [comment]
Yes, I would correct the issue but I don't know the reason of moving log init into main.py.
I tried the above fix locally.

**Comment by frisso** [comment]
Have you tried to ask suggestions @gabrielecastellano or @rbrtbnfgl ?

**Comment by rbrtbnfgl** [comment]
Did it drop some exceptions calling that method?

**Comment by gabrielecastellano** [comment]
Do you have the same problem also in the frog4-orchestrator?

**Comment by ReliableLion** [comment]
No there are no exceptions.
No, the frog4-orchestrator is able to init and print the log.

**Comment by gabrielecastellano** [comment]
However the code is the same in both repository. Could you check your configuration file (log section), comparing it with the frog-orchestrator one?

**Comment by ReliableLion** [comment]
The log sections are the same. I also checked that the log file variable is correctly retrieved by the SDN-DO
